testing/run_tests.py

```diff
@@ -565,12 +566,12 @@ def main():
   parser.add_argument('--use-sanitizer-suppressions', dest='sanitizer_suppressions', action='store_true',
       default=False, help='Provide the sanitizer suppressions lists to the via environment to the tests.')
   parser.add_argument('--adb-path', dest='adb_path', action='store',
-      default=None, help='Provide the path of adb used for android tests.  By default it looks on $PATH.')
+      default=None, help='Provide the path of adb used for android tests. By default it looks on $PATH.')
 
   args = parser.parse_args()
 
   if args.type == 'all':
-    types = ['engine', 'dart', 'benchmarks', 'java', 'objc', 'font-subset']
+    types = all_types
```

Comment:
This isn't identical to the previous version. The list now contains `android`.

Reply:
> The doc changes lgtm. But this changes the default behavior of the script. Not sure if this is the cause of the CI failure. But I agree, this all should include `android`.

Yes it changes the behavior, and I think it should include `android` into all, so I add the it. 

The CI failure is in shell_unittests, seems like it's not related to android tests(android test is an apk installed into the real device). Let me try the CI checking again.